src/App.jsx

```diff
@@ -1,15 +1,24 @@
 import React from 'react';
 import './App.scss';
+import Sum from './components/Sum';
 
-const App = () => (
+const App = (a, b) => (
   <>
-    <p>Sum of 2 and 3 is 5</p>
-    <p>Sum of -5 and 5 is 0</p>
-    <p>Sum of 10 and 0 is 10</p>
-    <p>Sum of 0 and 5 is 5</p>
-    <p>Sum of 0 and 0 is 0</p>
-    {/* Replace paragraphs with Sum componets */}
-    {/* And remove comment lines :) */}
+    <Sum 
+      a={2}
+      b={3}
+    />
```

Comment:
```suggestion
    <Sum a={2} b={3} />
```
There are not so many props here so you may write them in one line